### PR TITLE
Fix #2655 - dont populate deleted pinned/causatives

### LINKED ISSUE #2655: master version issue (not in prod): attempting to fetch classification from missing variant?
2021-05-24 10:12:23,542 - scout.server.app - ERROR: Exception on /cust000/F0005756-mip7 [GET] [in /home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py:1891]
Traceback (most recent call last):
 File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
   response = self.full_dispatch_request()
 File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
   rv = self.handle_user_exception(e)
 File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
   reraise(exc_type, exc_value, tb)
 File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
   raise value
 File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
   rv = self.dispatch_request()
 File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
   return self.view_functions[rule.endpoint](**req.view_args)
 File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/utils.py", line 26, in decorated_function
   context = func(*args, **kwargs)
 File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/blueprints/cases/views.py", line 66, in case
   data = controllers.case(store, institute_obj, case_obj)
 File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/blueprints/cases/controllers.py", line 96, in case
   _populate_acmg(causatives)
 File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/blueprints/cases/controllers.py", line 244, in _populate_acmg
   if isinstance(variant.get("acmg_classification"), int):
AttributeError: 'str' object has no attribute 'get'

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

**How to test**:
1. For a case with pinned or causative deleted / no longer loaded variants, Scout should no longer crash

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

Screenshot 2021-05-24 at 13 29 53

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

File: scout/server/blueprints/cases/controllers.py [hunk 1]
```diff
@@ -240,8 +240,10 @@ def _populate_acmg(evaluated_variants):
     Returns:
 
     """
+
     for variant in evaluated_variants:
-        if isinstance(variant.get("acmg_classification"), int):
+
+        if not isinstance(variant, str) and isinstance(variant.get("acmg_classification"), int):
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/controllers.py:246]: Nice! Let's put in on stage, we could try with the same variant that crashed!

**Review by northwestwitch — APPROVED** [review]
👍🏻

File: scout/server/blueprints/cases/controllers.py (hunk 1 above)
Comment (dnil) [reply scout/server/blueprints/cases/controllers.py:246]: Its there already! 😁

**Comment by northwestwitch** [comment]
Looks good. I've tested: https://scout-stage.scilifelab.se/cust000/F0005756-mip7 and https://scout-stage.scilifelab.se/cust000/F0009130-mip7

**Pull request merged**